README.md

```diff
@@ -501,8 +501,8 @@ Options for target 'cheribsd':
                         will break if not all dependencies have been built. (default: 'the value of the global
                         --freebsd-subdir options')
   --cheribsd/kernel-config CONFIG
-                        The kernel configuration to use for `make buildkernel` (default: CHERI_MALTA64 or
-                        CHERI128_MALTA64 depending on --cheri-bits) (default: 'target-dependent default')
+                        The kernel configuration to use for `make buildkernel` (default: CHERI_MALTA64)
```

Comment:
Should the MIPS-specific default clause just be removed entirely?

Reply:
Yes, I think that is probably best. It made sense a long time ago but no longer does.

Reply:
Trimmed.